src/components/JobActionsPopover.vue

```diff
@@ -54,8 +53,11 @@ export default defineComponent({
     })
   },
   methods: {
-    closePopover() {
-      popoverController.dismiss({ dismissed: true });
+    closePopover(enumId?: any) {
+      popoverController.dismiss({
+        dismissed: true,
```

Comment:
I think there is no need to pass `dismissed: true`

Reply:
Removed sir